_posts/newsletters/2019-10-24-newsletter.md

```diff
@@ -0,0 +1,155 @@
+---
+layout: post
+title: October 2019 US-RSE Newsletter
+subtitle: US-RSE Newsletter
```

Comment:
Maybe for title we should do “RSE Newsletter” and then for the subtitle have the date?

Reply:
Good suggestion.  It will look better on the archive page too.